### PR TITLE
Document get top 10 books review

### PR DESCRIPTION
**Pull request by at-minhdao**
## Description
- Get top 10 books review. With image, book name and score review
## Reference Requirements
- Product Backlog ID: PBI-35
- Task ID: #35.1

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:
- None

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by at-trampham — COMMENTED** [review]
LGTM

File: docs/api-references/book.md [hunk 1]
```diff
@@ -161,3 +161,103 @@ Get list books with paginate
...
+/api/books/top-review
+```
+Get top 10 books review
+#### Response - Success
+| Field | Type | Description |
+|-------|------|-------------|
+| meta | Object | Object meta |
+| status | String | Status result |
+| code | Number | HTTP status codes |
+| data | Object | Object book |
```
Comment (at-duyho) [inline docs/api-references/book.md]: - an Array or an object?

**Review by ghost — CHANGES_REQUESTED** [review]
check my comment!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**